sale_order_import/wizard/sale_order_import.py

```diff
@@ -263,16 +263,30 @@ def _prepare_order(self, parsed_order, price_source):
             so_vals["partner_invoice_id"] = invoicing_partner.id
         if parsed_order.get("date"):
             so_vals["date_order"] = parsed_order["date"]
+
+        errored_line = []
         for line in parsed_order["lines"]:
-            # partner=False because we don't want to use product.supplierinfo
-            product = bdio._match_product(
-                line["product"], parsed_order["chatter_msg"], seller=False
-            )
-            uom = bdio._match_uom(line.get("uom"), parsed_order["chatter_msg"], product)
-            line_vals = self._prepare_create_order_line(
-                product, uom, so_vals, line, price_source
-            )
-            so_vals["order_line"].append((0, 0, line_vals))
+            try:
+                # partner=False because we don't want to use product.supplierinfo
+                product = bdio._match_product(
+                    line["product"], parsed_order["chatter_msg"], seller=False
+                )
+                uom = bdio._match_uom(
+                    line.get("uom"), parsed_order["chatter_msg"], product
+                )
+                line_vals = self._prepare_create_order_line(
+                    product, uom, so_vals, line, price_source
+                )
+                so_vals["order_line"].append((0, 0, line_vals))
+            except Exception:
+                errored_line.append(str(line) + "<br/>")
```

Comment:
but we can't log a bit more precisely what was the error?

Reply:
I have updated with your suggestions

Reply:
No I mean log for the user, in the chatter

Reply:
Hi Nils, new update

![image](https://github.com/QuocDuong1306/edi/assets/127363167/a66bf199-b1ff-4638-af1c-fed5135030c0)